command/node_status.go

```diff
@@ -299,6 +299,16 @@ func nodeDrivers(n *api.Node) []string {
 	return drivers
 }
 
+func nodeVolumes(n *api.Node) []string {
```

Comment:
Maybe call it `nodeVolumeNames`?  also I liked how `outputNodeVolumeInfo` used `name` as the variable in the loop.

Reply:
Maintaining the same style as `nodeDrivers` for this. We can do a bigger refactor of this command at some point, but this probably isn't the right place for it.

Reply:
I would suggest for going for better clarity in new code.  Matchingold code ambiguity and tech debt feels like an odd justification for writing non-ideal new code tbqh.